# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../wxwidgets/pch-hotfix.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./contrib/include/wx/mmedia/vidbase.h.orig	2008-11-30 18:42:36.000000000 +0100
+++ ./contrib/include/wx/mmedia/vidbase.h	2008-11-30 18:39:20.000000000 +0100
@@ -16,7 +16,7 @@
 // headers
 // ----------------------------------------------------------------------------
 // For compilers that support precompilation, includes "wx/wx.h".
-#include "wx/wxprec.h"
+//#include "wx/wxprec.h"
 
 #ifdef __BORLANDC__
     #pragma hdrstop
--- ./contrib/include/wx/mmedia/vidxanm.h.orig	2008-11-30 18:42:42.000000000 +0100
+++ ./contrib/include/wx/mmedia/vidxanm.h	2008-11-30 18:41:03.000000000 +0100
@@ -15,7 +15,7 @@
 // headers
 // ----------------------------------------------------------------------------
 // For compilers that support precompilation, includes "wx/wx.h".
-#include "wx/wxprec.h"
+//#include "wx/wxprec.h"
 
 #ifdef __BORLANDC__
     #pragma hdrstop
